assets/js/components/notifications/FirstPartyModeSetupSuccessSubtleNotification.js

```diff
@@ -31,10 +31,26 @@ import { __ } from '@wordpress/i18n';
  */
 import SubtleNotification from '../../googlesitekit/notifications/components/layout/SubtleNotification';
 import Dismiss from '../../googlesitekit/notifications/components/common/Dismiss';
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../../googlesitekit/constants';
+import { NOTIFICATION_AREAS } from '../../googlesitekit/notifications/datastore/constants';
 
 export const FIRST_PARTY_MODE_SETUP_SUCCESS_NOTIFICATION =
 	'setup-success-notification-fpm';
 
+export const notification = [
+	FIRST_PARTY_MODE_SETUP_SUCCESS_NOTIFICATION,
+	{
+		Component: FirstPartyModeSetupSuccessSubtleNotification,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		isDismissible: false,
+		checkRequirements: () => {
+			return true;
+		},
```

Comment:
`checkRequirements` is optional and will be treated the same way if not provided
https://github.com/google/site-kit-wp/blob/1f927c21128bfa2846486dd1e393c83c882d0f68/assets/js/googlesitekit/notifications/datastore/notifications.js#L377-L382